### ISSUE TITLE
Client release process did not upload correct version to S3

### ISSUE THREAD

**Issue by robertknight** [hypothesis/client#602]
Client releases are hosted on https://cdn.hypothes.is, which is a backed by an S3 bucket. During the release build process, the client is first published to npm and then [s3-npm-publish](https://github.com/nickstenning/s3-npm-publish/blob/master/publish) is run to publish the "latest" client release to S3.

When v1.54 was tagged, the Jenkins build published the package as v1.54 to npm but apparently uploaded v1.53 to S3: https://jenkins.hypothes.is/job/client/job/master/568/console

We currently invoke `s3-npm-publish` without specifying a package name explicitly and my hypothesis is that npm is only eventually consistent with respect to updating the "latest" version of a package when a new released is published.

**Comment by robertknight** [comment]
I fixed the problem for this release by running the npm => S3 publishing tool manually.

```sh
git clone https://github.com/nickstenning/s3-npm-publish
cd s3-npm-publish
export AWS_ACCESS_KEY_ID=...
export AWS_SECRET_ACCESS_KEY=...
./publish hypothesis@1.54.0 s3://cdn.hypothes.is
```

**Comment by robertknight** [comment]
This happened again while @lyzadanger and me were working through the 1.62 release.

**Comment by robertknight** [comment]
#677 tentatively fixes this and the [v1.70](https://jenkins.hypothes.is/job/client/job/master/632/) release was successful. However I can't yet be sure that was luck or whether this definitely resolves the issue with eventual consistency within the npm registry after a release happens.
